docs/commands/preview-docs.md

```diff
@@ -1,39 +1,111 @@
+--------------------------|:---------:|:------------:|:-----------:|------------
+`entrypoint`              | `string`  | yes          | -           | Path to the API definition filename that you want to generate preview for. (Refer to [the entrypoints section below](#entrypoints) for more options)
+`--config`                | `string`  | no           | -           | Specify path to the config file
+`--force`, `-f`           | `boolean` | no           | -           | Generate preview output even when errors occur
+`--help`                  | `boolean` | no           | -           | Show help
+`--port`, `-p`            | `number`  | no           | 8080        | Preview port
+`--skip-decorator`        | `array`   | no           | -           | Ignore [certain decorators](#skip-preprocessor-or-decorator)
+`--skip-preprocessor`     | `array`   | no           | -           | Ignore [certain preprocessors](#skip-preprocessor-or-decorator)
+`--use-community-edition` | `boolean` | no           | -           | Force using Redoc Community Edition for docs preview
+`--version`               | `boolean` | no           | -           | Show version number
+
+## Examples
 
-```shell
-Positionals:
-  entrypoint                                                 [string] [required]
+### Entrypoints
 
-Options:
-  --version                Show version number.                        [boolean]
-  --help                   Show help.                                  [boolean]
-  --port, -p               Preview port.                [number] [default: 8080]
-  --skip-preprocessor      Ignore certain preprocessors.                 [array]
-  --skip-decorator         Ignore certain decorators.                    [array]
-  --use-community-edition  Force using Redoc CE for docs preview.      [boolean]
-  --force, -f              Produce bundle output even when errors occur.
-                                                                       [boolean]
-  --config                 Specify path to the config file.             [string]
+The command behaves differently depending on how you pass a path to the entrypoint to it and whether the [configuration file](#custom-configuration-file) exists.
+
+#### Pass entrypoint directly
+
+```bash
+openapi preview-docs openapi/openapi.yaml
 ```
 
+In this case, `preview-docs` will preview the definition that was passed to the command. The configuration file is ignored.
+
+#### Pass entrypoint without extension
+
+You can omit entrypoint's file extension when executing the `preview-docs` command. In this way, you can reference either `.yaml` or `.json` file.
 
-### How to preview the docs on a custom port
+```bash
+# preview-docs will preview either petstore.yaml or petstore.json file in the current working directory
+openapi preview-docs petstore
+# preview-docs will preview either sandbox.yaml or sandbox.json file in the openapi/extra directory
+openapi preview-docs openapi/extra/sandbox
+```
```

Comment:
This does not work. I wonder where this came from?

Proof:

```
➜  openapi preview-docs openapi/storefront

  🔎  Preview server running at http://127.0.0.1:8080

Bundling...

Failed to resolve entrypoint definition at openapi/storefront:

  - ENOENT: no such file or directory, open '/Users/adam/Projects/Rebilly/api-definitions/openapi/storefront'.


  👀  Watching openapi/storefront and all related resources for changes

^C
➜   openapi preview-docs openapi/storefront.yaml

  🔎  Preview server running at http://127.0.0.1:8080

Bundling...


  👀  Watching openapi/storefront.yaml and all related resources for changes

Created a bundle for openapi/storefront.yaml with warnings
```

Notice: `ENOENT: no such file or directory, open '/Users/adam/Projects/Rebilly/api-definitions/openapi/storefront'.`

Reply:
It's strange because I can't make it work too now. How then did I manage to do that before?! I will delete that section